src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,11 +5,21 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private String buldozerField;
+
     public Bulldozer() {
     }
 
     @Override
     public void doWork() {
         System.out.println("Bulldozer started to work");
     }
+
+    public String getBuldozerField() {
```

Comment:
The getter method name `getBuldozerField` contains a typo. It should be `getBulldozerField` to match the corrected field name.